src/Microsoft.Extensions.Configuration.AzureAppConfiguration/Models/LocalConfigurationClient.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+//
+
+using Azure.Data.AppConfiguration;
+using System;
+
+namespace Microsoft.Extensions.Configuration.AzureAppConfiguration.Models
+{
+    internal class LocalConfigurationClient
+    {
+        public LocalConfigurationClient(Uri Endpoint, ConfigurationClient client)
```

Comment:
Is null or null/empty valid?

Reply:
This constructor is guaranteed to receive a valid `endpoint` because it is not exposed to public, and we do validations before we call this constructor.

Reply:
Even though it's an internal class it's still a public method. Argument validation is still valuable. The reader can't see within this type that there is a guaranteed expectation of non-null.